### ISSUE TITLE
Camera motion and doorbell notifications no longer working

### ISSUE THREAD

**Issue by jordanmerrick** [dgreif/ring#445]
I have a Ring alarm, two stick-up cams, and a peephole doorbell. I've recently stopped receiving Home notifications for the doorbell and motion detected by any camera. The video feeds work, the contact and motion sensors for the alarm all trigger notifications fine, it's just the cameras and doorbell. 

HomeBridge logs even show when a camera has detected motion and/or a doorbell press occurred so I'm not sure where the problem is. Everything seems to be working and nothing has changed but camera-specific notifications don't seem to work anymore.

**Comment by dgreif** [comment]
Are you using a beta version of iOS or tvOS? If not, check your notification settings for the camera in the Home app

**Comment by jordanmerrick** [comment]
No beta software. Notifications are enabled for doorbell and motion on the doorbell camera, motion for others. 

![6E2D6B47-47A9-4EAC-A14F-2E71447A029F](https://user-images.githubusercontent.com/1397671/92186980-5407bc00-ee0d-11ea-99ab-faf14c5551a8.png)

**Comment by dgreif** [comment]
Can you open up the accessories within the camera in the Home app, and watch the motion sensor to see it if triggers when you move in front of the camera?  If it does, I'm not sure what else would be causing the issue. You could post some logs showing the motion being detected, which may give some insight. Also, you could try rebooting your phone and Apple TV or HomePod

**Comment by jordanmerrick** [comment]
I think I've found the cause though don't actually know what caused it or what this function does. I separated each of the cameras into tiles so I could see if motion was detected. Motion was detected but still no notification. I checked the notification settings for the motion sensors of each camera and there was an extra toggle for "Custom notification". I've never seen this before and it was enabled. 

I turned it off and notifications have worked fine since. I've reverted to showing single tiles and everything still seems to work.
